modules/product/src/main/java/com/opengamma/strata/product/etd/EtdFutureSecurity.java

```diff
@@ -123,6 +128,15 @@ public Trade createTrade(TradeInfo tradeInfo, double quantity, double tradePrice
         .build();
   }
 
+  /**
+   * Summarizes this ETD future into string form.
+   *
+   * @return the summary description
+   */
+  public String summaryDescription() {
+    return expiry.format(YM_FORMAT) + variant.getCode();
```

Comment:
Seems odd that the summary description for a future security doesn't include some form of contract identifier?

Reply:
The full description includes the ID - "<ID> x <quantity>, <etddescription>. So this is only part of the story.